invenio_rdm_records/services/config.py

```diff
@@ -293,6 +308,31 @@ class RDMRecordServiceConfig(RecordServiceConfig, ConfiguratorMixin):
                 }
             ),
         ),
+        # Parent
+        "parent": RecordLink(
+            "{+api}/records/{+parent_id}",
+            when=is_record,
+            vars=lambda record, vars: vars.update(
+                {"parent_id": record.parent.pid.pid_value}
+            ),
+        ),
+        "parent_html": RecordLink(
+            "{+ui}/records/{+parent_id}",
+            when=is_record,
+            vars=lambda record, vars: vars.update(
+                {"parent_id": record.parent.pid.pid_value}
+            ),
+        ),
+        "parent_doi": Link(
+            "{+ui}/doi/{+pid_doi}",
+            when=is_record_and_has_parent_doi,
+            vars=lambda record, vars: vars.update(
+                {
+                    f"pid_{scheme}": pid["identifier"]
+                    for (scheme, pid) in record.parent.pids.items()
+                }
+            ),
+        ),
```

Comment:
All these links depend on https://github.com/inveniosoftware/invenio-rdm-records/issues/1282 in order to work.